core/src/main/java/amino/run/kernel/server/KernelServerImpl.java

```diff
@@ -483,6 +483,7 @@ public static void main(String args[]) {
 
             // Register against OMS
             ServerInfo srvInfo = createServerInfo(host, ksArgs.labels);
+            srvInfo.processorCount = Runtime.getRuntime().availableProcessors();
```

Comment:
It's not clear to me why this initialization is not done in createServerInfo like all the other initialization?
Or better still, in the constructor of ServerInfo?
I'm uncomfortable with having an uninitialized processorCount in ServerInfo structure until someone remembers to initialize it.